contracts/erc2o/include/erc2o/erc2o.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <eosio/asset.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/singleton.hpp>
+#include <intx/intx.hpp>
+#include <erc2o/types.hpp>
+
+using namespace eosio;
+using namespace intx;
+
+namespace erc2o {
+
+class [[eosio::contract]] erc2o : public contract {
```

Comment:
We do not need to call this the `erc2o` contract. The reason for using `o` instead of `0` was because the only digits that Antelope names support are 1 to 5. But that is for the name of the account that will host this contract, not for the name of the contract itself. And remember you should not hard-code the `eosio.erc2o` name in this contract since that is not the best practice and also we will likely want to change that name at some point before we complete the advanced trustless bridge. 

We have the same flexibility as other C++ identifiers for naming the contract. So you could just call this contract `erc20`.

Reply:
Changed to erc20 everywhere.